src/Response/SapiStreamEmitter.php

```diff
@@ -24,9 +24,10 @@ class SapiStreamEmitter implements EmitterInterface
      * body content via the output buffer.
      *
      * @param ResponseInterface $response
+     * @param null|int $maxBufferLevel Maximum output buffering level to unwrap.
      * @param int $maxBufferLength Maximum output buffering size for each iteration
      */
-    public function emit(ResponseInterface $response, $maxBufferLength = 8192)
+    public function emit(ResponseInterface $response, $maxBufferLevel = null, $maxBufferLength = 8192)
```

Comment:
Okay, after looking at `Server.php`, `SapiEmitter.php`, and `SapiStreamEmitter.php`, I finally see what's going on. `Server` calls `emit()` with the response and buffer level, which is fine for the `SapiEmitter`, but cause problems in the `SapiStreamEmitter`, as that emitter is expecting the buffer length.

More problematic is we're passing this value, but the `EmitterInterface::emit()` definition only defines a single argument. I have a vague recollection that when we added output buffering awareness to the `SapiEmitter`, we'd already released a stable 1.0, which is why we made the argument optional.

I'm okay with this change, but will schedule it for 1.5.0 so we can draw attention to the breakage. I'll add tests when merging, as I have an idea on how to do it.